TextEditor.cpp

```diff
@@ -84,7 +84,7 @@ std::string TextEditor::GetText(const Coordinates & aStart, const Coordinates &
 	auto iend = GetCharacterIndex(aEnd);
 	size_t s = 0;
 
-	for (size_t i = lstart; i < lend; i++)
+	for (size_t i = lstart; (int)i < lend; i++)
```

Comment:
Don't you think it will be better to change the type returned by `GetCharacterIndex` instead ?

Using a C style cast `(int)i` make the compilation warning disappear but the problem overflow problem remains.